src/HealthChecks.DocumentDb/DocumentDbHealthCheck.cs

```diff
@@ -13,6 +13,8 @@ public DocumentDbHealthCheck(DocumentDbOptions documentDbOptions)
     {
         _documentDbOptions.UriEndpoint = Guard.ThrowIfNull(documentDbOptions.UriEndpoint);
         _documentDbOptions.PrimaryKey = Guard.ThrowIfNull(documentDbOptions.PrimaryKey);
+        _documentDbOptions.DatabaseName = documentDbOptions.DatabaseName;

```

Comment:
@cieciurm Also please remove copying values and just assign argument to field. I see no value in newing `DocumentDbOptions` in field and then copy all values from ctor's argument.

Reply:
It's been changed 😄

Reply:
I meant to store single options field.